Clarify min_ack_delay (#129)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

min_ack_delay (0xff03de1a):

: A variable-length integer representing the minimum amount of time in

>   microseconds by which the endpoint that is sending this vlaue is able to

Reviewer: Typo: /value/

>   microseconds by which the endpoint that is sending this vlaue is able to

Author: ```suggestion
  microseconds by which the endpoint that is sending this value is able to
```

>   delay an acknowledgement. This limit could be based on the receiver's clock

Reviewer: /limit/ ? I suggest /value/
... OR if you prefer /this value could be limited by .../

  or timer granularity. This information is used by the peer for selecting a

>   suitable value in the Request Max Ack Delay field in the ACK_FREQUENCY frame.

Reviewer: /in the ACK_FREQUENCY frame/of an ACK_FREQUENCY frame/ to avoid many /in/.

Author: ```suggestion
  suitable value in the Request Max Ack Delay field of the ACK_FREQUENCY frame.
```

>   or timer granularity. This information is used by the peer for selecting a
>   suitable value in the Request Max Ack Delay field of the ACK_FREQUENCY frame.

Reviewer 2: ```suggestion
  or timer granularity. 'min_ack_delay' is used by the peer to avoid requesting too
  small a value in the Request Max Ack Delay field of the ACK_FREQUENCY frame.
```
